src/marks/tip.js

```diff
@@ -200,8 +200,9 @@ export class Tip extends Mark {
         }
       }
       const line = selection.append("tspan").attr("x", 0).attr("dy", `${lineHeight}em`);
-      line.append("tspan").attr("font-weight", "bold").text(name);
+      if (name) line.append("tspan").attr("font-weight", "bold").text(name);
       if (value) line.append(() => document.createTextNode(value));
+      if (!name && !value) line.append(() => document.createTextNode(" "));
```

Comment:
```suggestion
      if (value || !name) line.append(() => document.createTextNode(value || "\u200b"));
```

Reply:
zwsp on its own doesn't create a line that is "meaty" enough to inform the height in all cases—the "after" test fails (both on Chrome and Safari) if we don't use a space.